src/features/user/Account/screens/index.tsx

```diff
@@ -0,0 +1,262 @@
+                null === filter ? styles.multiSelectInputCheckTrue : ''
+              }`}
+              key="all"
+              onClick={() => handleSetFilter(null)}
+            >
+              {t('all')}
+            </button>
+            {accountingFilters && (
+              <FilterButtons
+                accountingFilters={accountingFilters}
+                handleSetFilter={handleSetFilter}
+                filter={filter}
+              />
+            )}
+          </div>
+          <div className={styles.contentContainer}>
+            <div className={styles.accountsContainer}>
+              {paymentHistory.items.length === 0 ? (
+                <div className={styles.notFound}>
+                  <TransactionIcon color={'#c5c5c5'} width={'50px'} />
+                  <p>{t('noRecords')}</p>
+                </div>
+              ) : (
+                paymentHistory?.items?.map((item, index) => {
+                  return <PaymentRecord record={item} index={index} />;
+                })
+              )}
+            </div>
+            <div className={styles.filterContainerDesktop}>
+              <div className={styles.filterHead}>
+                <p className={styles.filterTitle}>{t('filters')}</p>
+              </div>
+              <div className={styles.filterGrid}>
+                <button
+                  className={`${styles.multiSelectInput} ${
+                    null === filter ? styles.multiSelectInputCheckTrue : ''
+                  }`}
+                  key="all"
+                  onClick={() => handleSetFilter(null)}
+                >
+                  {t('all')}
+                </button>
+                {accountingFilters && (
+                  <FilterButtons
+                    accountingFilters={accountingFilters}
+                    handleSetFilter={handleSetFilter}
+                    filter={filter}
+                  />
+                )}
+              </div>
+            </div>
+          </div>
+        </div>
+      )}
+      {/* Open setting component */}
+      {settingsModalOpen && (
+        <SettingsModal
+          userprofile={userprofile}
+          settingsModalOpen={settingsModalOpen}
+          handleSettingsModalClose={handleSettingsModalClose}
+          editProfileModalOpen={editProfileModalOpen}
+          handleEditProfileModalClose={handleEditProfileModalClose}
+          handleEditProfileModalOpen={handleEditProfileModalOpen}
+          changeForceReload={changeForceReload}
+          forceReload={forceReload}
+        />
+      )}
+    </div>
+  );
+}
+
+function FilterButtons(props: any) {
+  return Object.entries(props.accountingFilters).map((item) => {
+    return (
+      <button
+        className={`${styles.multiSelectInput} ${
+          props.filter === item[0] ? styles.multiSelectInputCheckTrue : ''
+        }`}
+        key={item[0]}
+        onClick={() => props.handleSetFilter(item[0])}
+      >
+        {item[0]}
```

Comment:
These items have to translations:

<img width="1003" alt="Bildschirmfoto 2021-04-05 um 12 26 46" src="https://user-images.githubusercontent.com/1532418/113564970-87f8c900-960a-11eb-83cb-4f3f973bb50b.png">

Reply:
Those values are coming from backend